Sanity/pkcs11/basic/runtest.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /CoreOS/clevis/Sanity/basic/pkcs11
+#   Description: tests the basic pkcs11 functionality of clevis
+#   Author: Martin Litwora <mlitwora@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2017 Red Hat, Inc.
```

Comment:
Wrong year (2024)

Reply:
fixed in the https://github.com/RedHat-SP-Security/clevis-tests/pull/8/commits/0711f411e56b6eca521842ef1fb393ceefd08eea